docs/toml-schema.md

```diff
@@ -301,13 +301,18 @@ octocat = "write"
 # The pattern matching the branches to be protected (required)
 pattern = "master"
 # Which CI checks to are required for merging (optional)
+# Cannot be set if `pr-required` is `false`.
 ci-checks = ["CI"]
 # Whether new commits after a reviewer's approval of a PR 
 # merging into this branch require another review. 
 # (optional - default `false`)
 dismiss-stale-review = false
+# Is a PR required when making changes to this branch?
```

Comment:
Should we consider consolidating all PR related options into a `pr-options` (or similar name)? The presence of the options key would be equivalent to `pr-required = true`.

Reply:
I wanted to do that originally, but it seemed a bit disruptive, since it would also need to contain `ci-checks`. Actually, pretty much all of the current fields except for `pattern` and `allowed-merge-teams` would need to be in the `pr-options` field. At that point it seemed easier to me to just keep it top level.